ui/src/app/pages/profile/data-use-agreement-panel.tsx

```diff
@@ -0,0 +1,159 @@
+      <DuaTextInput style={{margin: '0 1ex'}}
+                    disabled
+                    value={profile.givenName + ' ' + profile.familyName}
+                    onChange={(v) => this.setState({name: v})}
+                    data-test-id='dua-name-input'/>
+      ("Authorized User") have
+      personally reviewed this data use agreement. I agree to follow each of the policies
+      and procedures it describes.
+    </div>
+    <div>By entering my initials next to each statement below, I agree to these terms:</div>
+    <InitialsAgreement onChange={(v) => this.setState({initialWork: v})} value={initialWork}>
+      My work may be logged, monitored, and audited by the <i>All of Us</i> Research Program to ensure
+      compliance with policies and procedures, as well as the demonstration project charges.
+    </InitialsAgreement>
+    <InitialsAgreement onChange={(v) => this.setState({initialName: v})} value={initialName}>
+      My name, affiliation, profile information, and research description will be made public.
+      My research description will be used by the <AoUTitle/> to provide
+      participants with meaningful information about the research being conducted.
+    </InitialsAgreement>
+    <InitialsAgreement onChange={(v) => this.setState({initialSanctions: v})}
+                       value={initialSanctions}>
+      Access granted by the program in accordance with this agreement is exclusively
+      for participation in <i>All of Us</i> demonstration projects. At the conclusion of my
+      participation in these activities, I understand that my access to the <i>All of Us</i> data
+      resources will be revoked and/or subject to customary access policies and procedures.
+    </InitialsAgreement>
+    <div><strong>I acknowledge that failure to comply with the terms of this agreement may result
+      in termination of my <AoUTitle/> account and/or other sanctions, including
+      but not limited to:</strong>
+      <IndentedUnorderedList>
+        <IndentedListItem>
+          <strong>posting my name and affiliation on a publicly-accessible list of violators, and</strong>
+        </IndentedListItem>
+        <IndentedListItem><strong>notifying the National Institutes of Health or other federal agencies of my
+          actions.</strong></IndentedListItem>
+      </IndentedUnorderedList>
+      <div><strong>I understand that failure to comply with these terms may also carry additional financial, legal, or other
+        repercussions.</strong>
+      </div>
+    </div>
+    <div>
+      {showSanctionModal &&
+      <SanctionModal onClose={() => this.setState({showSanctionModal: false})}/>}
+    </div>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
+                  disabled value={this.props.profileState.profile.username}/>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
+                  disabled value={this.props.profileState.profile.contactEmail}/>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
```

Comment:
The google doc has labels on these input fields. not sure if they're needed though?